ansible_collections/arista/avd/roles/eos_designs/schemas/eos_designs.schema.yml

```diff
@@ -839,6 +839,71 @@ keys:
     description: Enable vlan aware bundles for EVPN MAC-VRF.
     type: bool
     default: false
+  evpn_vlan_bundles:
+    documentation_options:
+      table: evpn-vlan-bundles
+    type: list
+    primary_key: name
+    convert_types:
+    - dict
```

Comment:
We don't need to convert for newer schema

Reply:
removed